bokehjs/src/coffee/common/tool_manager.coffee

```diff
@@ -99,7 +99,7 @@ class ToolManager extends HasProperties
           continue
 
         gestures[et].tools.push(tool)
-        @listenTo(tool, 'change:active', _.bind(@_active_change, tool))
+        @listenTo(tool, 'change:active', () => @_active_change(tool))
```

Comment:
the scope of tool isn't what we want here I guess. Putting _.bind back (with the lambda taking a tool param) would be a possible clunky fix maybe.